service/controller/azurecluster/handler/azureconfig/credential.go

```diff
@@ -3,8 +3,8 @@ package azureconfig
 import (
 	"context"
 
-	"github.com/giantswarm/apiextensions/v3/pkg/apis/provider/v1alpha1"
-	apiextensionslabels "github.com/giantswarm/apiextensions/v3/pkg/label"
+	"github.com/giantswarm/apiextensions/v6/pkg/apis/provider/v1alpha1"
+	apiextensionslabels "github.com/giantswarm/apiextensions/v6/pkg/label"
```

Comment:
not important at all at this point, but this could be replaced by `k8smetadata` here